sources/install/package_desktop.sh

```diff
@@ -14,7 +14,7 @@ function install_xfce() {
     fapt terminator firefox-esr iproute2
 
     # Dependencies
-    fapt tigervnc-standalone-server tigervnc-xorg-extension tigervnc-viewer novnc websockify xfce4 dbus-x11 intltool libtool
+    fapt tigervnc-standalone-server tigervnc-xorg-extension tigervnc-viewer novnc websockify xfce4 dbus-x11 intltool libtool tigervnc-tools
```

Comment:
why adding `tigervnc-tools`?

Reply:
new packge in Bookworm for vnc tool (here you need the inster to have the `vncpasswd` command)

Reply:
Are you sure that `tigervnc-standalone-server` or `tigervnc-tools` doesn't install `vncpasswd` ? On Debian11, which package was it in ?

Reply:
tigervnc-tools is a new package in Bookworm